packages/interface/src/utils/fetchTally.ts

```diff
@@ -0,0 +1,54 @@
+import { config } from "~/config";
+
+import type { Tally } from "./types";
+
+import { createCachedFetch } from "./fetch";
+
+const cachedFetch = createCachedFetch({ ttl: 1000 * 60 * 10 });
+
+export interface GraphQLResponse {
+  data?: {
+    tally: Tally;
+  };
+}
+
+const tallyQuery = `
+query TallyQuery {
+    tally (id: $id) {
+      id
+      results {
+        id
+        result
+      }
+    }
+}
+`;
+
+/**
+ * Maps the tally data to the ITallyData interface
+ *
+ * @param tally - The tally data
+ * @returns The mapped tally data
+ */
+function mappedTallyData(tally: Tally): Tally {
+  return {
+    id: tally.id,
+    results: tally.results,
+  };
+}
```

Comment:
why we need to map `Tally` to `Tally` since they are actually the same?

Reply:
Thanks, I don't think it's needed anymore